library/include/ck/library/utility/check_err.hpp

```diff
@@ -178,7 +178,8 @@ check_err(const Range& out,
 template <typename Range, typename RefRange>
 std::enable_if_t<(std::is_same_v<ranges::range_value_t<Range>, ranges::range_value_t<RefRange>> &&
                   std::is_integral_v<ranges::range_value_t<Range>> &&
-                  !std::is_same_v<ranges::range_value_t<Range>, bhalf_t>)
+                  !std::is_same_v<ranges::range_value_t<Range>, bhalf_t> &&
+                  !std::is_same_v<ranges::range_value_t<Range>, f8_t>)
```

Comment:
@AngryLoki, thanks for adding this! Could you also add a similar check for bf8_t?

Reply:
Updated. Also removed mention of possible libc++ bug. As discussed there, libc++ conforms to standard, maybe libstdc++ should be changed to support Clang extended integer types, but not libc++.